include/crm/common/cmdline_internal.h

```diff
@@ -153,6 +153,10 @@ gchar *pcmk__quote_cmdline(gchar **argv);
  * \param[in] argv    The command line arguments.
  * \param[in] special Single-letter command line arguments that take a value.
  *                    These letters will all have pre-processing applied.
+ *
+ * \note @COMPAT We should drop this at a new major or minor release series
```

Comment:
I'm confused. The only internal callers are daemons and CLI tools. Booth depends on Pacemaker but not vice-versa. So how does Booth interact with the internal function `pcmk__cmdline_preproc()`.

Is Booth calling the Pacemaker CLI tools using this format? If so, maybe give details so we know where to look

Reply:
Yes, booth <1.2 uses the getopt syntax to invoke Pacemaker CLI tools

Reply:
Updated comment to be clearer

Reply:
Thanks. Although I misread the initial comment -- I didn't realize it was already fixed in the main branch of booth and it's just a version issue at this point. I was thinking it was like the sbd situation.